src/components/page/Home.js

```diff
@@ -41,30 +41,43 @@ export default function Home(props) {
     let durationStyle = {
         backgroundImage: "none",
         color: "#615CCD",
-        border: "2px solid #111"
+        border: "2px solid #111",
     };
 
     let supportedChainIds = [];
-    if(config.supportedChainArrray && config.supportedChainArrray.length > 0) {
-        config.supportedChainArrray.forEach((chain)=>
+    if (config.supportedChainArrray && config.supportedChainArrray.length > 0) {
+        config.supportedChainArrray.forEach((chain) =>
             supportedChainIds.push(chain.chainId)
         );
     }
 
     return (
-        <div className={classes.root}>
+        <Container maxWidth="xl">
```

Comment:
@tomarsachin2271 here I have added a `Container` to restrict the max width of the content and make it center aligned.